lib/matplotlib/backends/backend_wx.py

```diff
@@ -768,6 +768,7 @@ def _onSize(self, evt):
         else:
             # variable size
             size = self.GetClientSize()
+            size.IncTo(self.GetMinSize()) # Do not allow size to become smaller than MinSize
```

Comment:
This won't pass flake8 once Travis is working again; two spaces before `#` and the line is too long.